chapters/publication.tex

```diff
@@ -99,91 +181,265 @@ \section{Collaborating on academic writing}
+It allows other researchers to validate the mechanical construction of your results,
+to investigate what other results might be obtained from the same population,
+and test alternative approaches to other questions.
+Therefore you should make clear in your study
+where and how data are stored, and how and under what circumstances it might be accessed.
+You do not always have to complete the data publication data yourself,
+as long as you cite or otherwise directly reference data that you cannot release.
+Even if you think your raw data is owned by someone else,
+in many cases you will have the right to release
+at least some subset of your analytical dataset or the indicators you constructed.
+Check with the data supplier or other professional about licensing questions,
+particularly your right to publish derivative materials.
+You should only directly publish data which is fully de-identified
+and, to the extent required to ensure reasonable privacy,
+potentially identifying characteristics are further masked or removed.
+In all other cases, you should contact an appropriate data catalog
+to determine what privacy and licensing options are available.
+
+Make sure you have a clear understanding of the rights associated with the data release
+and communicate them to any future users of the data.
+You must provide a license with any data release.\sidenote{
+  \url{https://iatistandard.org/en/guidance/preparing-organisation/organisation-data-publication/how-to-license-your-data/}}
+This document need not be extremely detailed,
+but it should clearly communicate to the reader what they are allowed to do with your data and
+how credit should be given and to whom in further work that uses it.
+Keep in mind that you may or may not own your data,
+depending on how it was collected,
+and the best time to resolve any questions about these rights
+is at the time that data collection or transfer agreements are signed.
+Even if you cannot release data immediately or publicly,
+there are often options to catalog or archive the data without open publication.
+These may take the form of metadata catalogs or embargoed releases.
+Such setups allow you to hold an archival version of your data
+which your publication can reference,
+as well as provide information about the contents of the datasets
+and how future users might request permission to access them
+(even if you are not the person who can grant that permission).
+They can also provide for timed future releases of datasets
+once the need for exclusive access has ended.
+
+Data publication should release the dataset in a widely recognized format.
+While software-specific datasets are acceptable accompaniments to the code
+(since those precise materials are probably necessary),
+you should also consider releasing generic datasets
+such as CSV files with accompanying codebooks,
+since these will be re-adaptable by any researcher.
+Additionally, when possible, you should also release
```

Comment:
when is it not possible? i think better to assume documentation should always be provided

Reply:
d3b5eb4a91139e42a183a8beeb3989ede43b3774